pkg/wallet/account.go

```diff
@@ -277,14 +277,21 @@ func NewAccountFromEncryptedWIF(wif string, pass string, scrypt keys.ScryptParam
 
 // ConvertMultisig sets a's contract to multisig contract with m sufficient signatures.
 func (a *Account) ConvertMultisig(m int, pubs []*keys.PublicKey) error {
-	if a.Locked {
-		return errors.New("account is locked")
-	}
 	if a.privateKey == nil {
 		return errors.New("account key is not available (need to decrypt?)")
 	}
-	var found bool
 	accKey := a.privateKey.PublicKey()
+	return a.ConvertMultisigEncrypted(accKey, m, pubs)
+}
+
+// ConvertMultisigEncrypted sets a's contract to an encrypted multisig contract with m sufficient signatures.
+// The encrypted private key is not modified and remains the same.
+// This ensures that the converted account can still be decrypted with the original password.
+func (a *Account) ConvertMultisigEncrypted(accKey *keys.PublicKey, m int, pubs []*keys.PublicKey) error {
+	if a.Locked {
+		return errors.New("account is locked")
+	}
```

Comment:
Likely shouldn't be here.

Reply:
Well, let's for now revert this change, because here we don't require private key decryption. So I think it should be allowed to convert locked account to multisig if no WIF provided.

Let's also wait for @rom

Reply:
Locks are all about spending, creating another account should be possible.